flask_app/mocks.py

```diff
@@ -0,0 +1,47 @@
+SUPPORTED_EXCHANGES = {
+  "Wealthsimple": "ws-crypto-api"
+}
+
+# mock params
+
+MOCK_LOGIN_PARAM = {
+  "exchange": "Wealthsimple",
+  "username": "me@wealthsimple.com",
+  "password": "amazing_api"
+}
+
+MOCK_BALANCES_PARAM = {
+  "user_id": "user-123"
+}
+
+MOCK_TRANSFER_PARAMS = {
+  "account_id": "non-registered-crypto-123",
+  "source_currency": "CAD",
+  "destination_currency": "ETH",
+  "source_amount": 100,
```

Comment:
I'm thinking we should use camel case to be consistent with Tally.
```suggestion
  "accountId": "non-registered-crypto-123",
  "sourceCurrency": "CAD",
  "destinationCurrency": "ETH",
  "sourceAmount": 100,
```

Reply:
Also, we will probably need to use a more canonical way of referencing the `destinationCurrency`. It's possible that there may be more than 1 token with an ETH symbol. For example, what if it's wrapped ETH on Polygon chain. I think we should use `chainId` and `tokenAddress`.

```suggestion
  "accountId": "non-registered-crypto-123",
  "sourceCurrency": "CAD",
  "chainId": 1,
  "tokenAddress": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE" #0xEe is the placeholder for the native currency of a respective blockchain. https://github.com/DePayFi/widgets/issues/57#issuecomment-1166229695
  "sourceAmount": 100,
```

Reply:
True, is `chainId` + `tokenAddress` a unique identifier for a token?